Functions/Mock.ps1

```diff
@@ -1634,14 +1634,18 @@ function Reset-ConflictingParameters {
     $parameters
 }
 
-$script:ConflictingParameterNames = @(
-    "PSEdition"
-)
+[System.Collections.ArrayList]$script:ConflictingParameterNames = @()
 
 function Get-ConflictingParameterNames {
-    $script:ConflictingParameterNames
+    foreach ($var in (& $script:SafeCommands['Get-Variable'])) {
+        if (($var.Options -band [System.Management.Automation.ScopedItemOptions]::Constant) -or ($var.Options -band [System.Management.Automation.ScopedItemOptions]::ReadOnly)) {
+            $null = $script:ConflictingParameterNames.Add($var.Name)
+        }
+    }
 }
 
+Get-ConflictingParameterNames
```

Comment:
Calling the function here would have a similar purpose as to call it from Pester.psm1 during module import. 

But having it run directly from Mock seems counterintuitive, at least from a developer perspective, since it is not common for functions to be invoked at the script level from any Functions script.

Please move to Pester.psm1